src/components/super-cursor.js

```diff
@@ -62,49 +86,70 @@ AFRAME.registerComponent("super-cursor", {
     }
 
     if (this.isGrabbing || !isIntersecting) {
-      const head = this.el.object3D;
-      head.getWorldPosition(this.origin);
-      head.getWorldDirection(this.direction);
       const distance = Math.min(
         Math.max(this.data.minDistance, this.currentDistance - this.currentDistanceMod),
         this.data.maxDistance
       );
       this.currentDistanceMod = this.currentDistance - distance;
```

Comment:
bit confused what this is doing here?

Reply:
this is the logic that allows you to scroll the mouse wheel to move the object closer or farther from you.